tests/unit/test_column_group.py

```diff
@@ -57,8 +61,10 @@ def test_nested_column_group():
     assert geo_country_user[0] == geo_country_user[1]  # US / userA
     assert geo_country_user[2] != geo_country_user[0]  # same user but in canada
 
+    # TODO: Sort out how to check this since >>'ing a list in to an op no longer works
+
     # make sure we get an exception if we nest too deeply (can't handle arbitrarily deep
     # nested column groups - and the exceptions we would get in operators like Categorify
     # are super confusing for users)
-    with pytest.raises(ValueError):
-        cats = [[country + "user"] + country + "user"] >> Categorify(encode_type="combo")
+    # with pytest.raises(ValueError):
+    #   cats = [[country + "user"] + country + "user"] >> Categorify(encode_type="combo")
```

Comment:
Yeah, there are actually two issues with this particular test. The straightforward case where you do `["a","b","c"] >> ops.SomeOp` shouldn't be hard to make work. This case in particular is tricky because of the interactions it requires between `ColumnSelector`, `WorkflowNode`, and strings. It's been a while since I looked at it so it's not fresh in my memory, but I can try to debug what's going on here again to talk more informatively about the constraints that made it difficult.

Reply:
I see what you mean - I agree that you shouldn't put much effort to make a special case like this work.  I think the vanilla column list is the case that is most likely to break users code.

Reply:
As it turns out, once the list rshift syntax was re-instated, this test just works. Somewhere in the middle of the commits in this PR, this test was broken by the removal of the old nesting check error, but it's fixed by the `ValueError` raised when you try to put a `ColumnGroup` inside a `ColumnSelector`.